patches/minecraft/net/minecraft/command/impl/TeleportCommand.java.patch

```diff
@@ -1,5 +1,17 @@
 --- a/net/minecraft/command/impl/TeleportCommand.java
 +++ b/net/minecraft/command/impl/TeleportCommand.java
+@@ -121,6 +_,11 @@
+    }
+ 
+    private static void func_201127_a(CommandSource p_201127_0_, Entity p_201127_1_, ServerWorld p_201127_2_, double p_201127_3_, double p_201127_5_, double p_201127_7_, Set<SPlayerPositionLookPacket.Flags> p_201127_9_, float p_201127_10_, float p_201127_11_, @Nullable TeleportCommand.Facing p_201127_12_) throws CommandSyntaxException {
++      net.minecraft.util.math.vector.Vector3d target = net.minecraftforge.event.ForgeEventFactory.onEntityTeleportCommand(p_201127_1_, p_201127_3_, p_201127_5_, p_201127_7_);
++      if (target == null) return;
++      p_201127_3_ = target.field_72450_a;
++      p_201127_5_ = target.field_72448_b;
++      p_201127_7_ = target.field_72449_c;
```

Comment:
Can this be traded for a cancelled event and a one-line version for the x-y-z values?
```java
if(!event) return;
p_201127_3_ = event.target.field_72450_a; p_201127_5_ = event.target.field_72448_b; p_201127_7_ = event.target.field_72449_c;
```

Reply:
I'll refactor this soon

Reply:
Fixed with 8b629ad.